### DISCUSSION TITLE
How do I check logs?

### DISCUSSION THREAD

**Discussion by newoski** [JonathanTreffler/backblaze-personal-wine-container#137] [Q&A]
I'm running into a file that keeps reuploading over and over. Trying to look at the logs, but I can't find them.

**Comment by newoski** [comment]
Anybody?

**Reply by 10Thirty** [reply]
on unraid the backblaze logs are in

/mnt/user/appdata/Backblaze_Personal_Backup/wine/drive_c/ProgramData/Backblaze/bzdata/bzlogs/

have a look at bztransmit folder at the bztransmit{day}.log

it will tell you why files are retrying uploading.

also have a look at [https://old.reddit.com/r/backblaze/comments/18e1cgm/is_your_backup_corrupt_please_check/](url) as it could be the same issue you are having.

**Comment by newoski** [comment]
Thanks so much!!
